src/app/integrations/edit-page/action-configure/action-configure.component.ts

```diff
@@ -92,6 +92,24 @@ export class IntegrationsConfigureActionComponent extends FlowPage
       onSave: () => {
         if (!this.lastPage || this.page >= this.lastPage) {
           // all done...
+          this.integrationSupport.fetchMetadata(
+            this.step.connection,
+            this.step.action,
+            this.step.configuredProperties || {},
+          )
+          .toPromise()
+          .then(response => {
+            log.debug('Response: ' + JSON.stringify(response, undefined, 2));
+            const definition: any = response['_body']
+              ? JSON.parse(response['_body'])
+              : undefined;
+            this.step.action.inputDataShape = definition.inputDataShape;
+            this.step.action.outputDataShape = definition.outputDataShape;
+          })
+          .catch(response => {
+            // TODO error handling
```

Comment:
So for this bit if you set this attribute in the class:

https://github.com/zregvart/syndesis-ui/blob/8cb007d9352a7b0fef9fd4b17e69d294a55ffdb3/src/app/integrations/edit-page/action-configure/action-configure.component.ts#L37

like so:

https://github.com/zregvart/syndesis-ui/blob/8cb007d9352a7b0fef9fd4b17e69d294a55ffdb3/src/app/integrations/edit-page/action-configure/action-configure.component.ts#L172-L179

that'll add the inline notification that's in the page already here ->

https://github.com/zregvart/syndesis-ui/blob/8cb007d9352a7b0fef9fd4b17e69d294a55ffdb3/src/app/integrations/edit-page/action-configure/action-configure.component.html#L66-L68

Reply:
Ah cool had no idea we were using inline notifications this way, was just going to use the PatternFly class and an check for showing/hiding tbh lol.